.travis.yml

```diff
@@ -39,6 +39,7 @@ matrix:
       - ./src/gradlew -p src
       before_script:
       - sudo mkdir -p /etc/orderly/web/ && sudo touch /etc/orderly/web/go_signal
+      - ./scripts/migrate-build.sh
```

Comment:
We were just using the master migration image in the Travis build, so tests that depended on the presence of the Admin role were failing. So have added this line to build the migrations image for this branch locally, and have added running the appropriate migrations image into the travis/run-dependencies.sh script: https://github.com/vimc/orderly-web/pull/147/files#diff-6458df945a905cf9ec447b6709d1cd67R23